priv/repo/migrations/20180531213205_add_revenue_amount_and_distribution_rate_to_impressions.exs

```diff
@@ -0,0 +1,10 @@
+defmodule CodeFund.Repo.Migrations.AddRevenueAmountAndDistributionRateToImpressions do
+  use Ecto.Migration
+
+  def change do
+    alter table(:impressions) do
+      add :revenue_amount, :decimal, precision: 10, scale: 4, default: 0.0000, null: false
```

Comment:
I would change the scale here to 5 instead of 4.

Example: 2.17 / 1000 = .00217

Reply:
2.17 / 1000 * 0.38 = lots of digits.. Maybe set to 12 or so